src/ios-deploy/ios-deploy.m

```diff
@@ -2191,8 +2191,10 @@ void device_callback(struct am_device_notification_callback_info *info, void *ar
             break;
         case ADNCI_MSG_DISCONNECTED:
         {
+            CFStringRef device_interface_name = get_device_interface_name(info->dev);
             CFStringRef device_uuid = AMDeviceCopyDeviceIdentifier(info->dev);
-            NSLogOut(@"[....] Disconnected %@", device_uuid);
+            NSLogOut(@"[....] Disconnected %@ from %@.", device_uuid, device_interface_name);
+            CFRelease(device_interface_name);
```

Comment:
No need to release this since it's a constant(we don't elsewhere). https://developer.apple.com/documentation/corefoundation/cfstr